client/l3extLIfP_service.go

```diff
@@ -369,3 +369,141 @@ func (sm *ServiceManager) ReadRelationl3extRsNdIfPolFromLogicalInterfaceProfile(
 	}
 
 }
+
+func (sm *ServiceManager) CreateRelationPIMRsIfPol(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"annotation": "orchestrator:terraform",
+				"tDn": "%s"	
+			}
+		}
+	}`, "pimRsIfPol", dn, tDn))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+	return nil
+}
+
+func (sm *ServiceManager) ReadRelationPIMRsIfPol(parentDn string) (interface{}, error) {
+	dnUrl := fmt.Sprintf("%s/%s/%s.json", models.BaseurlStr, parentDn+"/"+models.RnPimIfP, "pimRsIfPol")
+	cont, err := sm.GetViaURL(dnUrl)
+	contList := models.ListFromContainer(cont, "pimRsIfPol")
+
+	if len(contList) > 0 {
+		dat := models.G(contList[0], "tDn")
+		return dat, err
+	} else {
+		return nil, err
+	}
+}
+
+func (sm *ServiceManager) DeleteRelationPIMRsIfPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsIfPol", parentDn)
+	return sm.DeleteByDn(dn, "pimRsIfPol")
+}
+
+func (sm *ServiceManager) CreateRelationPIMIPv6RsIfPol(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsV6IfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"annotation": "orchestrator:terraform",
+				"tDn": "%s"	
+			}
+		}
+	}`, "pimRsV6IfPol", dn, tDn))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+	return nil
+}
+
+func (sm *ServiceManager) ReadRelationPIMIPv6RsIfPol(parentDn string) (interface{}, error) {
+	dnUrl := fmt.Sprintf("%s/%s/%s.json", models.BaseurlStr, parentDn+"/"+models.RnPimIPV6IfP, "pimRsV6IfPol")
```

Comment:
other option would be
```suggestion
	dnUrl := fmt.Sprintf("%s/%s/%s/%s.json", models.BaseurlStr, parentDn, models.RnPimIPV6IfP, "pimRsV6IfPol")
```

Reply:
ok